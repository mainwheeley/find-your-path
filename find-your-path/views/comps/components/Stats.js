import React from 'react';
import {
    AppRegistry,
    Button,
    StyleSheet,
    Text,
    View,
    ListView,
    Image,
    Dimensions,
    ScrollView,
    Alert,
} from 'react-native';

export default class Stats extends React.Component {
    deletePrompt() {
	Alert.alert(
	    'Delete Account',
	    'Are you sure you want to delete your account?',
	    [
		{text: 'Delete', onPress: () => console.log('User delete')},
		{text: 'No', onPress: () => console.log('No delete')},
	    ],
	);
    }

  render() {
    return (

	<ScrollView>
	<View>
	<View style={styles.statsRow}>
	    <View style={styles.statsGrid}>
		<View style={styles.photoWrap}>
		    <Image style={styles.photo} resizeMode="contain" source={require('../imgs/run.png')} />
		</View>
		<View style={styles.textWrap}>
		    <Text style={styles.statsList}>Average pace: 7.5 min/mile</Text>
		</View>
	    </View>
	</View>

	<View style={styles.statsRow}>
	    <View style={styles.lineSeparator}></View>
	</View>

	<View style={styles.statsRow}>
	    <View style={styles.statsGrid}>
		<View style={styles.photoWrap}>
		    <Image style={styles.photo} resizeMode="stretch" source={require('../imgs/road.jpg')} />
		</View>
		<View style={styles.textWrap}>
		    <Text style={styles.statsList}>Total distance: 20 mi </Text>
		</View>
	    </View>
	</View>

	<View style={styles.statsRow}>
	    <View style={styles.lineSeparator}></View>
	</View>

	<View style={styles.statsRow}>
	    <View style={styles.statsGrid}>
		<View style={styles.photoWrap}>
		    <Image style={styles.photo} resizeMode="contain" source={require('../imgs/fire.jpg')} />
		</View>
		<View style={styles.textWrap}>
		    <Text style={styles.statsList}>Ave. cal burned: 150 cals</Text>
		</View>
	    </View>
	</View>

	<View style={styles.statsRow}>
	    <View style={styles.lineSeparator}></View>
	</View>

	<View style={styles.statsRow}>
	    <View style={styles.statsGrid}>
		<View style={styles.photoWrap}>
		    <Image style={styles.photo} resizeMode="contain" source={require('../imgs/time.png')} />
		</View>
		<View style={styles.textWrap}>
		    <Text style={styles.statsList}>Total time run: 12 hrs</Text>
		</View>
	    </View>
	</View>

	<View style={styles.statsRow}>
	    <View style={styles.lineSeparator}></View>
	</View>

  <View style={styles.statsRow}>
      <View style={styles.statsGrid}>
    <View style={styles.photoWrap}>
        <Image style={styles.photo} resizeMode="contain" source={require('../imgs/star.png')} />
    </View>
    <View style={styles.textWrap}>
        <Text style={styles.statsList}>My Popular Routes</Text>
    </View>
      </View>
  </View>

  <View style={styles.statsRow}>
      <View style={styles.statsGrid}>
    <View style={styles.photoWrap}>
        <Image style={styles.photo} resizeMode="contain" source={require('../imgs/one.jpg')} />
    </View>
    <View style={styles.textWrap}>
        <Text style={styles.statsListRoute}>134 W State St.</Text>
        <Text style={styles.statsListDescription}>Total Distance: 6 mi</Text>
        <Text style={styles.statsListDescription}># of times ran: 4</Text>
        <Text style={styles.statsListDescription}>Specified Dest: Trails</Text>
        <Text style={styles.statsListDescription}>Workout Type: Bike</Text>
    </View>
      </View>
  </View>

  <View style={styles.statsRow}>
      <View style={styles.statsGrid}>
    <View style={styles.photoWrap}>
        <Image style={styles.photo} resizeMode="contain" source={require('../imgs/two.jpg')} />
    </View>
    <View style={styles.textWrap}>
        <Text style={styles.statsListRoute}>610 Purdue Mall</Text>
        <Text style={styles.statsListDescription}>Total Distance: 5 mi</Text>
        <Text style={styles.statsListDescription}># of times ran: 3</Text>
        <Text style={styles.statsListDescription}>Specified Dest: Beach</Text>
        <Text style={styles.statsListDescription}>Workout Type: Run</Text>
    </View>
      </View>
  </View>

  <View style={styles.statsRow}>
      <View style={styles.statsGrid}>
    <View style={styles.photoWrap}>
        <Image style={styles.photo} resizeMode="contain" source={require('../imgs/three.jpg')} />
    </View>
    <View style={styles.textWrap}>
        <Text style={styles.statsListRoute}>230 S Grant St.</Text>
        <Text style={styles.statsListDescription}>Total Distance: 4 mi</Text>
        <Text style={styles.statsListDescription}># of times ran: 2</Text>
        <Text style={styles.statsListDescription}>Specified Dest: Park</Text>
        <Text style={styles.statsListDescription}>Workout Type: Walk</Text>
    </View>
      </View>
  </View>

  <View style={styles.statsRow}>
      <View style={styles.lineSeparator}></View>
  </View>

  <View style={styles.statsRow}>
      <View style={styles.statsGrid}>
    <View style={styles.photoWrap}>
        <Image style={styles.photo} resizeMode="contain" source={require('../imgs/heart.jpg')} />
    </View>
    <View style={styles.textWrap}>
        <Text style={styles.statsList}>Fav. route: 134 W State St.</Text>
    </View>
      </View>
  </View>

	</View>
	</ScrollView>
    );
  }
}

const styles = StyleSheet.create({
    delButton: {
	flex: 1,
	height: (Dimensions.get('window').width / 5) - 25,
	alignItems: 'center',
	justifyContent: 'center',
    },
    lineSeparator: {
	backgroundColor: '#000',
	height: 1,
	width: (Dimensions.get('window').width - 25),
	alignSelf: 'center',
    },
    boxTest: {
	backgroundColor: '#e3aa1a',
    },
    statsList: {
	fontSize: 20,
	fontWeight: 'bold',
	textAlignVertical: 'center',
    },
    statsListRoute: {
      fontSize: 18,
      fontWeight: 'bold',
      textAlignVertical: 'center',
    },
    statsListDescription: {
      fontSize: 13,
      fontWeight: 'bold',
      textAlignVertical: 'center',
    },
    statsGrid: {
	flex: 1,
	flexDirection: 'row',
	flexWrap: 'wrap',
    },
    statsRow: {
	flex: 1,
	flexDirection: 'column',
	flexWrap: 'wrap',
    },
    photoWrap: {
	margin: 10,
	height: (Dimensions.get('window').width / 5),
	width: (Dimensions.get('window').width / 5),
	//width: 75,
    },
    textWrap: {
	alignItems: 'center',
	justifyContent: 'center',
    },
    photo: {
	flex: 1,
	height: (Dimensions.get('window').width / 5) - 25,
	width: (Dimensions.get('window').width / 5) - 25,
	alignSelf: 'stretch',
//	alignItems: 'center',
//	alignSelf: 'stretch',
    },
});
