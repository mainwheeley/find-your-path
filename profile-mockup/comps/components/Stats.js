import React from 'react';
import { 
    AppRegistry,
    StyleSheet,
    Text,
    View,
    ListView,
    Image,
    Dimensions,
    ScrollView
} from 'react-native';

export default class Stats extends React.Component {
  render() {
    return (

	<ScrollView>
	<View>
	<View style={styles.statsRow}>
	    <View style={styles.statsGrid}>
		<View style={styles.photoWrap}>
		    <Image style={styles.photo} resizeMode="contain" source={require('../imgs/run.png')} />
		</View>
		<View style={styles.textWrap}>
		    <Text style={styles.statsList}>Average pace: </Text>
		</View>
	    </View>
	</View>

	<View style={styles.statsRow}>
	    <View style={styles.lineSeparator}></View>
	</View>

	<View style={styles.statsRow}>
	    <View style={styles.statsGrid}>
		<View style={styles.photoWrap}>
		    <Image style={styles.photo} resizeMode="stretch" source={require('../imgs/road.jpg')} />
		</View>
		<View style={styles.textWrap}>
		    <Text style={styles.statsList}>Total distance: </Text>
		</View>
	    </View>
	</View>

	<View style={styles.statsRow}>
	    <View style={styles.lineSeparator}></View>
	</View>

	<View style={styles.statsRow}>
	    <View style={styles.statsGrid}>
		<View style={styles.photoWrap}>
		    <Image style={styles.photo} resizeMode="contain" source={require('../imgs/fire.jpg')} />
		</View>
		<View style={styles.textWrap}>
		    <Text style={styles.statsList}>Total calories burned: </Text>
		</View>
	    </View>
	</View>

	<View style={styles.statsRow}>
	    <View style={styles.lineSeparator}></View>
	</View>

	<View style={styles.statsRow}>
	    <View style={styles.statsGrid}>
		<View style={styles.photoWrap}>
		    <Image style={styles.photo} resizeMode="contain" source={require('../imgs/time.png')} />
		</View>
		<View style={styles.textWrap}>
		    <Text style={styles.statsList}>Total time run: </Text>
		</View>
	    </View>
	</View>

	<View style={styles.statsRow}>
	    <View style={styles.lineSeparator}></View>
	</View>

	<View style={styles.statsRow}>
	    <View style={styles.statsGrid}>
		<View style={styles.photoWrap}>
		    <Image style={styles.photo} resizeMode="contain" source={require('../imgs/star.png')} />
		</View>
		<View style={styles.textWrap}>
		    <Text style={styles.statsList}>Favorite route: </Text>
		</View>
	    </View>
	</View>

	</View>
	</ScrollView>
    );
  }
}

const styles = StyleSheet.create({
    lineSeparator: {
	backgroundColor: '#000',
	height: 1,
	width: (Dimensions.get('window').width - 25),
	alignSelf: 'center',
    },
    boxTest: {
	backgroundColor: '#e3aa1a',
    },
    statsList: {
	fontSize: 20,
	fontWeight: 'bold',
	textAlignVertical: 'center',
    },
    statsGrid: {
	flex: 1,
	flexDirection: 'row',
	flexWrap: 'wrap',
    },
    statsRow: {
	flex: 1,
	flexDirection: 'column',
	flexWrap: 'wrap',
    },
    photoWrap: {
	margin: 10,
	height: (Dimensions.get('window').width / 5),
	width: (Dimensions.get('window').width / 5),
	//width: 75,
    },
    textWrap: {
	alignItems: 'center',
	justifyContent: 'center',
    },
    photo: {
	flex: 1,
	height: (Dimensions.get('window').width / 5) - 25,
	width: (Dimensions.get('window').width / 5) - 25,
	alignSelf: 'stretch',
//	alignItems: 'center',
//	alignSelf: 'stretch',
    },
});

