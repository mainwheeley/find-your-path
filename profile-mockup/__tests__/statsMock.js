import React from 'react';
import { View, Text, StyleSheet } from 'react-native';
import Stats from '../comps/components/Stats';
import { shallow } from 'enzyme';
describe('Profile Bar', function() {
    it('renders correctly', () => {
	const tree = shallow(<Stats />)
	expect(tree).toMatchSnapshot();
    });
    it('length 1', function() {
	const tree = shallow(<Stats />)
	expect(tree.length).toBe(1);
    });
});
