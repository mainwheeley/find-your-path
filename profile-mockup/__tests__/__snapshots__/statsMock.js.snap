// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile Bar renders correctly 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <ScrollViewMock>
    <View>
      <View
        style={
          Object {
            "flex": 1,
            "flexDirection": "column",
            "flexWrap": "wrap",
          }
        }
      >
        <View
          style={
            Object {
              "flex": 1,
              "flexDirection": "row",
              "flexWrap": "wrap",
            }
          }
        >
          <View
            style={
              Object {
                "height": 150,
                "margin": 10,
                "width": 150,
              }
            }
          >
            <Image
              resizeMode="contain"
              source={1}
              style={
                Object {
                  "alignSelf": "stretch",
                  "flex": 1,
                  "height": 125,
                  "width": 125,
                }
              }
            />
          </View>
          <View
            style={
              Object {
                "alignItems": "center",
                "justifyContent": "center",
              }
            }
          >
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
              style={
                Object {
                  "fontSize": 20,
                  "fontWeight": "bold",
                  "textAlignVertical": "center",
                }
              }
            >
              Average pace: 7.5 min/mile
            </Text>
          </View>
        </View>
      </View>
      <View
        style={
          Object {
            "flex": 1,
            "flexDirection": "column",
            "flexWrap": "wrap",
          }
        }
      >
        <View
          style={
            Object {
              "alignSelf": "center",
              "backgroundColor": "#000",
              "height": 1,
              "width": 725,
            }
          }
        />
      </View>
      <View
        style={
          Object {
            "flex": 1,
            "flexDirection": "column",
            "flexWrap": "wrap",
          }
        }
      >
        <View
          style={
            Object {
              "flex": 1,
              "flexDirection": "row",
              "flexWrap": "wrap",
            }
          }
        >
          <View
            style={
              Object {
                "height": 150,
                "margin": 10,
                "width": 150,
              }
            }
          >
            <Image
              resizeMode="stretch"
              source={1}
              style={
                Object {
                  "alignSelf": "stretch",
                  "flex": 1,
                  "height": 125,
                  "width": 125,
                }
              }
            />
          </View>
          <View
            style={
              Object {
                "alignItems": "center",
                "justifyContent": "center",
              }
            }
          >
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
              style={
                Object {
                  "fontSize": 20,
                  "fontWeight": "bold",
                  "textAlignVertical": "center",
                }
              }
            >
              Total distance: 200 mi 
            </Text>
          </View>
        </View>
      </View>
      <View
        style={
          Object {
            "flex": 1,
            "flexDirection": "column",
            "flexWrap": "wrap",
          }
        }
      >
        <View
          style={
            Object {
              "alignSelf": "center",
              "backgroundColor": "#000",
              "height": 1,
              "width": 725,
            }
          }
        />
      </View>
      <View
        style={
          Object {
            "flex": 1,
            "flexDirection": "column",
            "flexWrap": "wrap",
          }
        }
      >
        <View
          style={
            Object {
              "flex": 1,
              "flexDirection": "row",
              "flexWrap": "wrap",
            }
          }
        >
          <View
            style={
              Object {
                "height": 150,
                "margin": 10,
                "width": 150,
              }
            }
          >
            <Image
              resizeMode="contain"
              source={1}
              style={
                Object {
                  "alignSelf": "stretch",
                  "flex": 1,
                  "height": 125,
                  "width": 125,
                }
              }
            />
          </View>
          <View
            style={
              Object {
                "alignItems": "center",
                "justifyContent": "center",
              }
            }
          >
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
              style={
                Object {
                  "fontSize": 20,
                  "fontWeight": "bold",
                  "textAlignVertical": "center",
                }
              }
            >
              Ave. calories burned: 150 cals
            </Text>
          </View>
        </View>
      </View>
      <View
        style={
          Object {
            "flex": 1,
            "flexDirection": "column",
            "flexWrap": "wrap",
          }
        }
      >
        <View
          style={
            Object {
              "alignSelf": "center",
              "backgroundColor": "#000",
              "height": 1,
              "width": 725,
            }
          }
        />
      </View>
      <View
        style={
          Object {
            "flex": 1,
            "flexDirection": "column",
            "flexWrap": "wrap",
          }
        }
      >
        <View
          style={
            Object {
              "flex": 1,
              "flexDirection": "row",
              "flexWrap": "wrap",
            }
          }
        >
          <View
            style={
              Object {
                "height": 150,
                "margin": 10,
                "width": 150,
              }
            }
          >
            <Image
              resizeMode="contain"
              source={1}
              style={
                Object {
                  "alignSelf": "stretch",
                  "flex": 1,
                  "height": 125,
                  "width": 125,
                }
              }
            />
          </View>
          <View
            style={
              Object {
                "alignItems": "center",
                "justifyContent": "center",
              }
            }
          >
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
              style={
                Object {
                  "fontSize": 20,
                  "fontWeight": "bold",
                  "textAlignVertical": "center",
                }
              }
            >
              Total time run: 200 hrs
            </Text>
          </View>
        </View>
      </View>
      <View
        style={
          Object {
            "flex": 1,
            "flexDirection": "column",
            "flexWrap": "wrap",
          }
        }
      >
        <View
          style={
            Object {
              "alignSelf": "center",
              "backgroundColor": "#000",
              "height": 1,
              "width": 725,
            }
          }
        />
      </View>
      <View
        style={
          Object {
            "flex": 1,
            "flexDirection": "column",
            "flexWrap": "wrap",
          }
        }
      >
        <View
          style={
            Object {
              "flex": 1,
              "flexDirection": "row",
              "flexWrap": "wrap",
            }
          }
        >
          <View
            style={
              Object {
                "height": 150,
                "margin": 10,
                "width": 150,
              }
            }
          >
            <Image
              resizeMode="contain"
              source={1}
              style={
                Object {
                  "alignSelf": "stretch",
                  "flex": 1,
                  "height": 125,
                  "width": 125,
                }
              }
            />
          </View>
          <View
            style={
              Object {
                "alignItems": "center",
                "justifyContent": "center",
              }
            }
          >
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
              style={
                Object {
                  "fontSize": 20,
                  "fontWeight": "bold",
                  "textAlignVertical": "center",
                }
              }
            >
              Favorite route: State St.
            </Text>
          </View>
        </View>
      </View>
    </View>
  </ScrollViewMock>,
  "nodes": Array [
    <ScrollViewMock>
      <View>
        <View
          style={
            Object {
              "flex": 1,
              "flexDirection": "column",
              "flexWrap": "wrap",
            }
          }
        >
          <View
            style={
              Object {
                "flex": 1,
                "flexDirection": "row",
                "flexWrap": "wrap",
              }
            }
          >
            <View
              style={
                Object {
                  "height": 150,
                  "margin": 10,
                  "width": 150,
                }
              }
            >
              <Image
                resizeMode="contain"
                source={1}
                style={
                  Object {
                    "alignSelf": "stretch",
                    "flex": 1,
                    "height": 125,
                    "width": 125,
                  }
                }
              />
            </View>
            <View
              style={
                Object {
                  "alignItems": "center",
                  "justifyContent": "center",
                }
              }
            >
              <Text
                accessible={true}
                allowFontScaling={true}
                disabled={false}
                ellipsizeMode="tail"
                style={
                  Object {
                    "fontSize": 20,
                    "fontWeight": "bold",
                    "textAlignVertical": "center",
                  }
                }
              >
                Average pace: 7.5 min/mile
              </Text>
            </View>
          </View>
        </View>
        <View
          style={
            Object {
              "flex": 1,
              "flexDirection": "column",
              "flexWrap": "wrap",
            }
          }
        >
          <View
            style={
              Object {
                "alignSelf": "center",
                "backgroundColor": "#000",
                "height": 1,
                "width": 725,
              }
            }
          />
        </View>
        <View
          style={
            Object {
              "flex": 1,
              "flexDirection": "column",
              "flexWrap": "wrap",
            }
          }
        >
          <View
            style={
              Object {
                "flex": 1,
                "flexDirection": "row",
                "flexWrap": "wrap",
              }
            }
          >
            <View
              style={
                Object {
                  "height": 150,
                  "margin": 10,
                  "width": 150,
                }
              }
            >
              <Image
                resizeMode="stretch"
                source={1}
                style={
                  Object {
                    "alignSelf": "stretch",
                    "flex": 1,
                    "height": 125,
                    "width": 125,
                  }
                }
              />
            </View>
            <View
              style={
                Object {
                  "alignItems": "center",
                  "justifyContent": "center",
                }
              }
            >
              <Text
                accessible={true}
                allowFontScaling={true}
                disabled={false}
                ellipsizeMode="tail"
                style={
                  Object {
                    "fontSize": 20,
                    "fontWeight": "bold",
                    "textAlignVertical": "center",
                  }
                }
              >
                Total distance: 200 mi 
              </Text>
            </View>
          </View>
        </View>
        <View
          style={
            Object {
              "flex": 1,
              "flexDirection": "column",
              "flexWrap": "wrap",
            }
          }
        >
          <View
            style={
              Object {
                "alignSelf": "center",
                "backgroundColor": "#000",
                "height": 1,
                "width": 725,
              }
            }
          />
        </View>
        <View
          style={
            Object {
              "flex": 1,
              "flexDirection": "column",
              "flexWrap": "wrap",
            }
          }
        >
          <View
            style={
              Object {
                "flex": 1,
                "flexDirection": "row",
                "flexWrap": "wrap",
              }
            }
          >
            <View
              style={
                Object {
                  "height": 150,
                  "margin": 10,
                  "width": 150,
                }
              }
            >
              <Image
                resizeMode="contain"
                source={1}
                style={
                  Object {
                    "alignSelf": "stretch",
                    "flex": 1,
                    "height": 125,
                    "width": 125,
                  }
                }
              />
            </View>
            <View
              style={
                Object {
                  "alignItems": "center",
                  "justifyContent": "center",
                }
              }
            >
              <Text
                accessible={true}
                allowFontScaling={true}
                disabled={false}
                ellipsizeMode="tail"
                style={
                  Object {
                    "fontSize": 20,
                    "fontWeight": "bold",
                    "textAlignVertical": "center",
                  }
                }
              >
                Ave. calories burned: 150 cals
              </Text>
            </View>
          </View>
        </View>
        <View
          style={
            Object {
              "flex": 1,
              "flexDirection": "column",
              "flexWrap": "wrap",
            }
          }
        >
          <View
            style={
              Object {
                "alignSelf": "center",
                "backgroundColor": "#000",
                "height": 1,
                "width": 725,
              }
            }
          />
        </View>
        <View
          style={
            Object {
              "flex": 1,
              "flexDirection": "column",
              "flexWrap": "wrap",
            }
          }
        >
          <View
            style={
              Object {
                "flex": 1,
                "flexDirection": "row",
                "flexWrap": "wrap",
              }
            }
          >
            <View
              style={
                Object {
                  "height": 150,
                  "margin": 10,
                  "width": 150,
                }
              }
            >
              <Image
                resizeMode="contain"
                source={1}
                style={
                  Object {
                    "alignSelf": "stretch",
                    "flex": 1,
                    "height": 125,
                    "width": 125,
                  }
                }
              />
            </View>
            <View
              style={
                Object {
                  "alignItems": "center",
                  "justifyContent": "center",
                }
              }
            >
              <Text
                accessible={true}
                allowFontScaling={true}
                disabled={false}
                ellipsizeMode="tail"
                style={
                  Object {
                    "fontSize": 20,
                    "fontWeight": "bold",
                    "textAlignVertical": "center",
                  }
                }
              >
                Total time run: 200 hrs
              </Text>
            </View>
          </View>
        </View>
        <View
          style={
            Object {
              "flex": 1,
              "flexDirection": "column",
              "flexWrap": "wrap",
            }
          }
        >
          <View
            style={
              Object {
                "alignSelf": "center",
                "backgroundColor": "#000",
                "height": 1,
                "width": 725,
              }
            }
          />
        </View>
        <View
          style={
            Object {
              "flex": 1,
              "flexDirection": "column",
              "flexWrap": "wrap",
            }
          }
        >
          <View
            style={
              Object {
                "flex": 1,
                "flexDirection": "row",
                "flexWrap": "wrap",
              }
            }
          >
            <View
              style={
                Object {
                  "height": 150,
                  "margin": 10,
                  "width": 150,
                }
              }
            >
              <Image
                resizeMode="contain"
                source={1}
                style={
                  Object {
                    "alignSelf": "stretch",
                    "flex": 1,
                    "height": 125,
                    "width": 125,
                  }
                }
              />
            </View>
            <View
              style={
                Object {
                  "alignItems": "center",
                  "justifyContent": "center",
                }
              }
            >
              <Text
                accessible={true}
                allowFontScaling={true}
                disabled={false}
                ellipsizeMode="tail"
                style={
                  Object {
                    "fontSize": 20,
                    "fontWeight": "bold",
                    "textAlignVertical": "center",
                  }
                }
              >
                Favorite route: State St.
              </Text>
            </View>
          </View>
        </View>
      </View>
    </ScrollViewMock>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_context": Object {},
    "_element": <Stats />,
    "_instance": Stats {
      "context": Object {},
      "props": Object {},
      "refs": Object {},
      "state": Object {},
      "updater": Updater {
        "_renderer": [Circular],
      },
    },
    "_newState": null,
    "_rendered": <ScrollViewMock>
      <View>
        <View
          style={
            Object {
              "flex": 1,
              "flexDirection": "column",
              "flexWrap": "wrap",
            }
          }
        >
          <View
            style={
              Object {
                "flex": 1,
                "flexDirection": "row",
                "flexWrap": "wrap",
              }
            }
          >
            <View
              style={
                Object {
                  "height": 150,
                  "margin": 10,
                  "width": 150,
                }
              }
            >
              <Image
                resizeMode="contain"
                source={1}
                style={
                  Object {
                    "alignSelf": "stretch",
                    "flex": 1,
                    "height": 125,
                    "width": 125,
                  }
                }
              />
            </View>
            <View
              style={
                Object {
                  "alignItems": "center",
                  "justifyContent": "center",
                }
              }
            >
              <Text
                accessible={true}
                allowFontScaling={true}
                disabled={false}
                ellipsizeMode="tail"
                style={
                  Object {
                    "fontSize": 20,
                    "fontWeight": "bold",
                    "textAlignVertical": "center",
                  }
                }
              >
                Average pace: 7.5 min/mile
              </Text>
            </View>
          </View>
        </View>
        <View
          style={
            Object {
              "flex": 1,
              "flexDirection": "column",
              "flexWrap": "wrap",
            }
          }
        >
          <View
            style={
              Object {
                "alignSelf": "center",
                "backgroundColor": "#000",
                "height": 1,
                "width": 725,
              }
            }
          />
        </View>
        <View
          style={
            Object {
              "flex": 1,
              "flexDirection": "column",
              "flexWrap": "wrap",
            }
          }
        >
          <View
            style={
              Object {
                "flex": 1,
                "flexDirection": "row",
                "flexWrap": "wrap",
              }
            }
          >
            <View
              style={
                Object {
                  "height": 150,
                  "margin": 10,
                  "width": 150,
                }
              }
            >
              <Image
                resizeMode="stretch"
                source={1}
                style={
                  Object {
                    "alignSelf": "stretch",
                    "flex": 1,
                    "height": 125,
                    "width": 125,
                  }
                }
              />
            </View>
            <View
              style={
                Object {
                  "alignItems": "center",
                  "justifyContent": "center",
                }
              }
            >
              <Text
                accessible={true}
                allowFontScaling={true}
                disabled={false}
                ellipsizeMode="tail"
                style={
                  Object {
                    "fontSize": 20,
                    "fontWeight": "bold",
                    "textAlignVertical": "center",
                  }
                }
              >
                Total distance: 200 mi 
              </Text>
            </View>
          </View>
        </View>
        <View
          style={
            Object {
              "flex": 1,
              "flexDirection": "column",
              "flexWrap": "wrap",
            }
          }
        >
          <View
            style={
              Object {
                "alignSelf": "center",
                "backgroundColor": "#000",
                "height": 1,
                "width": 725,
              }
            }
          />
        </View>
        <View
          style={
            Object {
              "flex": 1,
              "flexDirection": "column",
              "flexWrap": "wrap",
            }
          }
        >
          <View
            style={
              Object {
                "flex": 1,
                "flexDirection": "row",
                "flexWrap": "wrap",
              }
            }
          >
            <View
              style={
                Object {
                  "height": 150,
                  "margin": 10,
                  "width": 150,
                }
              }
            >
              <Image
                resizeMode="contain"
                source={1}
                style={
                  Object {
                    "alignSelf": "stretch",
                    "flex": 1,
                    "height": 125,
                    "width": 125,
                  }
                }
              />
            </View>
            <View
              style={
                Object {
                  "alignItems": "center",
                  "justifyContent": "center",
                }
              }
            >
              <Text
                accessible={true}
                allowFontScaling={true}
                disabled={false}
                ellipsizeMode="tail"
                style={
                  Object {
                    "fontSize": 20,
                    "fontWeight": "bold",
                    "textAlignVertical": "center",
                  }
                }
              >
                Ave. calories burned: 150 cals
              </Text>
            </View>
          </View>
        </View>
        <View
          style={
            Object {
              "flex": 1,
              "flexDirection": "column",
              "flexWrap": "wrap",
            }
          }
        >
          <View
            style={
              Object {
                "alignSelf": "center",
                "backgroundColor": "#000",
                "height": 1,
                "width": 725,
              }
            }
          />
        </View>
        <View
          style={
            Object {
              "flex": 1,
              "flexDirection": "column",
              "flexWrap": "wrap",
            }
          }
        >
          <View
            style={
              Object {
                "flex": 1,
                "flexDirection": "row",
                "flexWrap": "wrap",
              }
            }
          >
            <View
              style={
                Object {
                  "height": 150,
                  "margin": 10,
                  "width": 150,
                }
              }
            >
              <Image
                resizeMode="contain"
                source={1}
                style={
                  Object {
                    "alignSelf": "stretch",
                    "flex": 1,
                    "height": 125,
                    "width": 125,
                  }
                }
              />
            </View>
            <View
              style={
                Object {
                  "alignItems": "center",
                  "justifyContent": "center",
                }
              }
            >
              <Text
                accessible={true}
                allowFontScaling={true}
                disabled={false}
                ellipsizeMode="tail"
                style={
                  Object {
                    "fontSize": 20,
                    "fontWeight": "bold",
                    "textAlignVertical": "center",
                  }
                }
              >
                Total time run: 200 hrs
              </Text>
            </View>
          </View>
        </View>
        <View
          style={
            Object {
              "flex": 1,
              "flexDirection": "column",
              "flexWrap": "wrap",
            }
          }
        >
          <View
            style={
              Object {
                "alignSelf": "center",
                "backgroundColor": "#000",
                "height": 1,
                "width": 725,
              }
            }
          />
        </View>
        <View
          style={
            Object {
              "flex": 1,
              "flexDirection": "column",
              "flexWrap": "wrap",
            }
          }
        >
          <View
            style={
              Object {
                "flex": 1,
                "flexDirection": "row",
                "flexWrap": "wrap",
              }
            }
          >
            <View
              style={
                Object {
                  "height": 150,
                  "margin": 10,
                  "width": 150,
                }
              }
            >
              <Image
                resizeMode="contain"
                source={1}
                style={
                  Object {
                    "alignSelf": "stretch",
                    "flex": 1,
                    "height": 125,
                    "width": 125,
                  }
                }
              />
            </View>
            <View
              style={
                Object {
                  "alignItems": "center",
                  "justifyContent": "center",
                }
              }
            >
              <Text
                accessible={true}
                allowFontScaling={true}
                disabled={false}
                ellipsizeMode="tail"
                style={
                  Object {
                    "fontSize": 20,
                    "fontWeight": "bold",
                    "textAlignVertical": "center",
                  }
                }
              >
                Favorite route: State St.
              </Text>
            </View>
          </View>
        </View>
      </View>
    </ScrollViewMock>,
    "_rendering": false,
    "_updater": Updater {
      "_renderer": [Circular],
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <Stats />,
}
`;
